server/cmwell-ws/app/controllers/Application.scala

```diff
@@ -2139,7 +2141,9 @@ callback=< [URL] >
         val xg = queryString.keySet("xg")
         val yg = queryString.keySet("yg")
 
-        if (offset > Settings.maxOffset) {
+        if (rawSortParamsTry.isFailure) {
```

Comment:
Nice fix! But I think it deserves its own PR. I don't see how it is related to STP fix.

Reply:
Yes it's unrelated.
Just got fixed along the way.